docs/introduction.md

```diff
@@ -7,65 +7,55 @@ title: Syllabus
 
 ## Course Application
 
-Apply [here](http://bit.ly/webdevsp20).
+Apply [here](https://bit.ly/web-dev-fa20).
 
 ## Introduction
 
 This class will teach students about modern web development technologies, practices, and industry standards to better equip them for academic work, research, interviews, internships, and full-time employment.
 
 ## When are office hours?
 
-Office hours will start **Monday, April 6th** and be administered remotely via Zoom. See below for office hour times and Zoom links:
+Office hours times are still being worked out. Check back here when the class starts for accurate Zoom links and times!
 
-Sam: Monday 3:30-4:30pm [Zoom Link](https://cornell.zoom.us/j/9389498164?pwd=cjI1Y1VBZGt6eTlBaWFQVnk0QTBrUT09)
-
-Peter: Monday 4:30-5:30pm [Zoom Link](https://cornell.zoom.us/j/2749196798?pwd=RldMRXFocjBLcWZTblVXcUU1Q0dXZz09)
-
-Megan: Tuesday 1pm-2pm [Zoom Link](https://cornell.zoom.us/j/7293410777)
-
-Enoch: Tuesday 2-3pm [Zoom Link](https://cornell.zoom.us/j/2233738601)
-
-Ashneel: Wednesday 6:45pm-7:45pm [Zoom Link](https://cornell.zoom.us/j/8798851713)
-
-Megan: Thursday 1pm-2pm [Zoom Link](https://cornell.zoom.us/j/7293410777)
-
-If you are unable to make any of those office hours times, you can ask questions on [Piazza](https://piazza.com/class/k6in32n6fckgk?cid=4#) or get in touch with Megan or Ashneel.
+If you are unable to make any of those office hours times, you can ask questions on Piazza (link will be added closer to course start) or get in touch with Megan or Ashneel.
 
 ## Is there a lecture schedule?
 
-Yes!
+Yes! This is a tentative schedule and topics covered are subject to change. Exact dates are subject to change as well based on Cornell guidance on the ongoing pandemic. Currently, we are hoping to finish the course before Thanksgiving break.
 
-Lecture 1 2/12/20: Syllabus, Intro to Node, Express Routes, HTTP Methods
+Lecture 0 9/15: Syllabus, JavaScript/TypeScript
 
-Lecture 2 2/19/20: NoSQL DB (Firebase)
+Lecture 1 9/22: Intro to Node, Express Routes, HTTP Methods
 
-Lecture 3 2/26/20: More Firebase, Build a Rest API for DB model
+Lecture 2 9/29: NoSQL DB (Firebase)
 
-Lecture 4 3/04/20: Functional Programming (map, filter, reduce)
+Lecture 3 10/06: More Firebase, Build a Rest API for DB model
 
-Lecture 5 3/11/20: React (Components and tooling)
+Lecture 4 10/13: Functional Programming (map, filter, reduce)
 
-Lecture 6 4/08/20: React (JSX and conditional rendering)
+Lecture 5 10/20: React (Components and tooling)
 
-Lecture 7 4/15/20: React (ES6 and State)
+Lecture 6 10/27: React (JSX and conditional rendering)
 
-Lecture 8 4/22/20: frontend + backend (Data Fetching)
+Lecture 7 11/3: React (ES6 and State)
 
-Lecture 9 4/29/20: Deployment
+Lecture 8 11/10: frontend + backend (Data Fetching)
 
-Lecture 10 5/06/20: Student Choice
+Lecture 9 11/17: Deployment & TypeScript
```

Comment:
not push blocking, but it looks like we should find something else to teach instead of TS in this lecture.